src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -5489,7 +5491,61 @@ public void testListNodes() throws Exception {
       assertThat(plugin.name(), allOf(notNullValue(), not("")));
     }
   }
-  
+
+  @Test
+  public void testInspectNode() throws Exception {
+    requireDockerApiVersionAtLeast("1.24", "swarm support");
+    List<Node> nodes = sut.listNodes();
+    assertThat(nodes.size(), greaterThanOrEqualTo(1));
+
+    Node nut = nodes.get(0);
+
+
+    assertThat(nut.id(), allOf(notNullValue(), not("")));
+
+    Date now = new Date();
+    NodeInfo nodeInfo = sut.inspectNode(nut.id());
+
+    assertThat(nodeInfo.id(), allOf(notNullValue(), not("")));
```

Comment:
For this and the next few lines, you don't need to assert that they are non-null. They are guaranteed to be non-null since the methods in the `NodeInfo` class were not annotated `@Nullable`.